### DISCUSSION TITLE
Scrum Prep Milestone 1 - Week 2

### DISCUSSION THREAD

**Discussion by PedroRodz** [uprm-inso4101-2024-2025-s1/semester-project-regiupr#41] [Scrum Preps]
Write your scrum prep for week 2 of milestone 1
Week: September 6 - September 12

**Comment by PedroRodz** [comment]
-Closed a research task 
-In the process of closing a coding task for the database team

**Comment by J3SSY-ANDU** [comment]
- Closed a research task for the database CRUD functions implementation
- Closed the students and student_courses database tables CRUD functions implementation

**Comment by carlospepin23** [comment]
- This week I worked with the research task on how to create and deploy a mySQL database on AWS.

- I collaborated in the task of creating and finalizing the database that is being hosted by an AWS Elastic Compute Cloud instance.

- I collaborated in the testing of the database, making sure it is accessible through Python scripts using the official mysql-connector-python library.

**Comment by Yamil-Serrano** [comment]
- The task of creating an executable test for Windows operative system has been closed.

- Finish the creation of the clases of Student and Courses for it data management.

**Comment by cristianramos9** [comment]
No work done. Busy with homework and studying for exam.

**Comment by cristianMarcial** [comment]
Defining tasks, not completed them
